awx/ui_next/src/components/AddRole/AddResourceRole.jsx

```diff
@@ -187,11 +187,13 @@ class AddResourceRole extends React.Component {
             <SelectableCard
               isSelected={selectedResource === 'users'}
               label={i18n._(t`Users`)}
+              dataCy="Add-Role-Users"
```

Comment:
Let's change these to `add-role-users`, `add-role-teams` to keep it consistent with the other `data-cy`'s.